german/Leichen/Simulation/Simulation.md

```diff
@@ -8,4 +8,6 @@ Nachdem ich vergeblich auf k
 angebrachten Gegenst�nden - wie einer VR-Brille oder verkabelten Handschuhen - gewartet habe,
 muss ich mich nach anderen Hinweisen umschauen.
 
-Deshalb erw�ge ich folgende Optionen:
+Deshalb erw�ge ich folgende Optionen
+
+... um zu sehen ob das Wasser real ist, entscheide ich mich erstmal ein bisschen zu [schwimmen](../../schwimmen/schwimmen.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.